storage/database/interface.go

```diff
@@ -42,7 +42,7 @@ func (db DBType) ToValid() DBType {
 
 	for _, vdb := range validDBType {
 		if strings.ToLower(string(vdb)) == strings.ToLower(string(db)) {
-			return db
+			return vdb
```

Comment:
It would be better if we have a simple test for this

Reply:
I added it. PTAL :)